src/core/lib/iomgr/tcp_server_uv.c

```diff
@@ -76,13 +76,30 @@ struct grpc_tcp_server {
 
   /* shutdown callback */
   grpc_closure *shutdown_complete;
+
```

Comment:
Ok, let's just leave it empty for now... there's plenty of docs in the related .h files.